tests/e2e/config/environment.js

```diff
@@ -23,4 +23,36 @@ class E2EEnvironment extends PuppeteerEnvironment {
 	}
 }
 
+// This code is useful for debug purposes.
```

Comment:
I assume you left this comments in on purpose, so that people could easily uncomment it to debug stuff, right?

If that's the case, could you write a bit more on the comment about why and how it is useful and how and when should people use this code?